prow/plugins/trigger/trigger.go

```diff
@@ -233,10 +240,25 @@ func runRequested(c Client, pr *github.PullRequest, requestedJobs []config.Presu
 		return err
 	}
 
+	cloneRef := fmt.Sprintf("%s/%s", pr.Base.Repo.Owner.Login, pr.Base.Repo.Name)
+	gitRepo, err := c.GitClient.Clone(cloneRef)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := gitRepo.Clean(); err != nil {
+			c.Logger.WithError(err).Error("Error cleaning up repo.")
+		}
+	}()
+	changedFiles, err := gitRepo.Diff(baseSHA, pr.Head.SHA)
```

Comment:
Don't you need to fetch the PR branch before using the PR SHA?

Reply:
Thank you for the review. Current version tries to address that using the `Clone` call before the `Diff`. Please let me know if that does not seem sufficient.